File: C++/70.cpp
class Solution {
public:    
    int climbStairs(int n) { 
        """
        :type: n: int
        :rtype: int
        """
        if (n == 1)
            return 1;
        else if (n == 2)
            return 2;
        else {
            vector<int> s(n, 0);
            s[0] = 1;
            s[1] = 2;
            for (int i = 2; i < n; i++) 
                s[i] = s[i-1] + s[i-2];
            return s[n-1];
        }
    }
};

File: Python/70.py
class Solution:
    def climbStairs(self, n):
        '''
        :type: n: int
        :rtype: int
        '''
        arr = [1, 2]
        for i in range(2, n):
            arr.append(arr[i-1]+arr[i-2])
        return arr[n-1]